calico-cloud/operations/cluster-management.mdx

```diff
@@ -16,15 +16,16 @@ If you inadvertenty delete a managed cluster, rerun the `curl` command and the c
 
 ### Access your managed clusters
 
-Each time you install {{prodname}} on a cluster, the cluster is added to the drop-down menu in Manager.
+Each time you install {{prodname}} on a cluster, the cluster is added to the drop-down menu in Manager UI.
 
 ![cluster-menu](/img/calico-cloud/cluster-menu.png)
 
-Use this menu to move between your managed clusters. You can create as many managed clusters as your subscriptions allows.
+Use this menu to move between your clusters. You can connect as many managed clusters as your subscriptions allows.
 
 ### Managed cluster views
 
-Each time you change to a different cluster, the entire Manager "view" is set for the cluster you've selected. For example, each managed cluster has unique indexes for Elasticsearch clusters, a separate instance of Kibana, and the dashboard reflects only the selected managed cluster.
+When you select a different cluster from the cluster menu, the entire Manager UI view changes to reflect the selected cluster.
+For example, each managed cluster has unique indexes for Elasticsearch clusters, a separate instance of Kibana, and the dashboard reflects only the selected managed cluster.
 
 ### Manage policies in multiple clusters
```

Comment:
Phil/Slide UX team said that highlighting UIs with color is not a good practice and asked that we stop using this as a callout method. Their usability studies show users believe color highlights are part of the UI. That's why I moved to something that was a non-UI element.

Reply:
Fair enough. I saw highlighting of this sort elsewhere, so I went with it.

The main idea here is how to change cluster views. Since the menu itself is dynamic, we can't easily describe the menu, and a focused screenshot, lacking the context of the rest of the UI, is also difficult to understand.

I agree with the principal of avoiding these callouts. But I'm unsure of what we could do better in this particular situation.

Any ideas?